src/App.tsx

```diff
@@ -1,61 +1,107 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const initialTodo = {
+    id: 0,
+    title: '',
+    completed: false,
+    userId: 0,
+  };
+
+  const [wasSubmitted, setWasSubmitted] = useState(false);
+  const [todos, setTodos] = useState(todosFromServer);
+  const [newTodo, setNewTodo] = useState(initialTodo);
+
+  const hasTitleError = wasSubmitted && newTodo.title === '';
+  const hasUserError = wasSubmitted && newTodo.userId === 0;
```

Comment:
just check it inside `handleSubmit` and get rid of `wasSubmitted` state
also, trim `title` during validation to prevent adding title with spaces only